vivisect/extensions/__init__.py

```diff
@@ -52,8 +54,15 @@ def loadExtensions(vw, vwgui):
                 module.vw = vw
                 spec.loader.exec_module(module)
 
+                if not hasattr(module, 'vivExtension'):
+                    logger.info("Skipping python module %r, not a valid extension", modpath)
```

Comment:
Seems like making these into `warning()` calls would be appropriate. If you've got things in your modules path that don't implement the convention :D

Reply:
```suggestion
                    logger.warning("Skipping python module %r, not a valid extension (module has no vivExtension() function)", modpath)
```